trashtalk/google_sheets.py

```diff
@@ -5,19 +5,14 @@
 
 import gspread
 import datetime
-from oauth2client.service_account import ServiceAccountCredentials
 
 from trashtalk.models import Cleanup, db_session
-from flask import Blueprint, render_template, request, flash, current_app
 
-scope = ['https://spreadsheets.google.com/feeds/'] #Sheet must be shared with Cloud account in order to find it
-validation = ""#Validation pulls from JSON account information file
-sheet_key = "" #Key from sheet URL
-top_row = 2 #First Row is header #Limited ability to find the end of the data. Easier just to put data at the top and find it later
-#
-# credentials = ServiceAccountCredentials.from_json_keyfile_name(validation, scope)
-# gc = gspread.authorize(credentials)
-# wks = gc.open_by_key(sheet_key).sheet1
+top_row = 2 #First Row is header: Limited ability to find the end of the data. Easier just to put data at the top and find it later
+
+credentials = ServiceAccountCredentials.from_json_keyfile_name(GOOGLE_SHEETS_VALIDATION, GOOGLE_SHEETS_SCOPE)
```

Comment:
I think you intended to use the config objects on `current_app` here? Where is `GOOGLE_SHEETS_VALIDATION` for example defined?

Reply:
Yup! I'll update it the next version with **config current app**